### ISSUE TITLE
No nightly since 2018-12-09

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/rust#56667]
I am not seeing new nightlies for the last two nights:
```
$ rustup update
info: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'
info: syncing channel updates for 'nightly-x86_64-unknown-linux-gnu'
info: checking for self-updates

   stable-x86_64-unknown-linux-gnu unchanged - rustc 1.31.0 (abe02cefd 2018-12-04)
  nightly-x86_64-unknown-linux-gnu unchanged - rustc 1.32.0-nightly (4a45578bc 2018-12-07)
```

Cc @rust-lang/infra

**Comment by Enet4** [comment]
This became particularly alarming because the latest nightly does not provide any of the `rls`, `rustfmt` and `clippy` preview components ([components history](https://mexus.github.io/rustup-components-history/)). I was trying to understand why, but this matter might just be out of my hands.

**Comment by kennytm** [comment]
Nominating for discussion.

See also the previous issue #56586.

**Comment by adals** [comment]
i can not update

$ rustup update
info: syncing channel updates for 'stable-x86_64-apple-darwin'
info: syncing channel updates for 'nightly-x86_64-apple-darwin'
info: latest update on 2018-12-08, rust version 1.32.0-nightly (4a45578bc 2018-12-07)
error: some components unavailable for download: 'clippy', 'rls', 'rustfmt'
info: checking for self-updates

       stable-x86_64-apple-darwin unchanged - rustc 1.31.0 (abe02cefd 2018-12-04)
  nightly-x86_64-apple-darwin update failed - rustc 1.32.0-nightly (14997d56a 2018-12-05)

**Comment by Mark-Simulacrum** [comment]
The dist machine was recently upgraded to Ubuntu 18.04; it seems that that upgrade also made us lose python by-default. I've committed https://github.com/rust-lang/rust-central-station/commit/c5b2d8d870b0f180177adef0dca03773d6d1622d and will try to deploy sometime today; hopefully that's the last case of bugs arising on RCS.

**Comment by alexcrichton** [comment]
> I'm honestly not quite sure, as I believe this tool is built on Rust's CI 

Heh this appears to be empirical evidence that we don't build it on CI!

In any case, [sent a PR for a fix](https://github.com/rust-lang/rust/pull/56703)

**Comment by temannin** [comment]
Not sure if this is related but issuing the command:

`curl -s https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly` 

is currently failing on my system

**Comment by kennytm** [comment]
Probably will be fixed by #56735.

**Comment by RalfJung** [comment]
Again, no nightly. Isn't it possible to manually trigger the nightly release during the day so that debugging doesn't have to wait an entire day each time?

**Comment by brokenthorn** [comment]
No `rustfmt` or `clippy` for `rustc 1.32.0-nightly (4a45578bc 2018-12-07)`. 😞

```
» rustup component list
cargo-x86_64-apple-darwin (default)
lldb-preview-x86_64-apple-darwin
llvm-tools-preview-x86_64-apple-darwin
rust-analysis-x86_64-apple-darwin
rust-docs-x86_64-apple-darwin (default)
rust-src
rust-std-aarch64-apple-ios
rust-std-aarch64-fuchsia
rust-std-aarch64-linux-android
rust-std-aarch64-pc-windows-msvc
rust-std-aarch64-unknown-cloudabi
rust-std-aarch64-unknown-linux-gnu
rust-std-aarch64-unknown-linux-musl
rust-std-arm-linux-androideabi
rust-std-arm-unknown-linux-gnueabi
rust-std-arm-unknown-linux-gnueabihf
rust-std-arm-unknown-linux-musleabi
rust-std-arm-unknown-linux-musleabihf
rust-std-armebv7r-none-eabi
rust-std-armebv7r-none-eabihf
rust-std-armv5te-unknown-linux-gnueabi
rust-std-armv5te-unknown-linux-musleabi
rust-std-armv7-apple-ios
rust-std-armv7-linux-androideabi
rust-std-armv7-unknown-linux-gnueabihf
rust-std-armv7-unknown-linux-musleabihf
rust-std-armv7r-none-eabi
rust-std-armv7r-none-eabihf
rust-std-armv7s-apple-ios
rust-std-asmjs-unknown-emscripten
rust-std-i386-apple-ios
rust-std-i586-pc-windows-msvc
rust-std-i586-unknown-linux-gnu
rust-std-i586-unknown-linux-musl
rust-std-i686-apple-darwin
rust-std-i686-linux-android
rust-std-i686-pc-windows-gnu
rust-std-i686-pc-windows-msvc
rust-std-i686-unknown-freebsd
rust-std-i686-unknown-linux-gnu
rust-std-i686-unknown-linux-musl
rust-std-mips-unknown-linux-gnu
rust-std-mips-unknown-linux-musl
rust-std-mips64-unknown-linux-gnuabi64
rust-std-mips64el-unknown-linux-gnuabi64
rust-std-mipsel-unknown-linux-gnu
rust-std-mipsel-unknown-linux-musl
rust-std-powerpc-unknown-linux-gnu
rust-std-powerpc64-unknown-linux-gnu
rust-std-powerpc64le-unknown-linux-gnu
rust-std-riscv32imac-unknown-none-elf
rust-std-riscv32imc-unknown-none-elf
rust-std-s390x-unknown-linux-gnu
rust-std-sparc64-unknown-linux-gnu
rust-std-sparcv9-sun-solaris
rust-std-thumbv6m-none-eabi
rust-std-thumbv7em-none-eabi
rust-std-thumbv7em-none-eabihf
rust-std-thumbv7m-none-eabi
rust-std-wasm32-unknown-emscripten
rust-std-wasm32-unknown-unknown
rust-std-x86_64-apple-darwin (default)
rust-std-x86_64-apple-ios
rust-std-x86_64-fuchsia
rust-std-x86_64-linux-android
rust-std-x86_64-pc-windows-gnu
rust-std-x86_64-pc-windows-msvc
rust-std-x86_64-rumprun-netbsd
rust-std-x86_64-sun-solaris
rust-std-x86_64-unknown-cloudabi
rust-std-x86_64-unknown-freebsd
rust-std-x86_64-unknown-linux-gnu
rust-std-x86_64-unknown-linux-gnux32
rust-std-x86_64-unknown-linux-musl
rust-std-x86_64-unknown-netbsd
rust-std-x86_64-unknown-redox
rustc-x86_64-apple-darwin (default)

» rustup show
Default host: x86_64-apple-darwin

nightly-x86_64-apple-darwin (default)
rustc 1.32.0-nightly (4a45578bc 2018-12-07)

**Comment by matthiaskrgr** [comment]
@brokenthorn The latest nightly that was released unfortunately has broken components.
https://mexus.github.io/rustup-components-history/
In the meantime, tools have been fixed again and should be available again as soon as a new nightly releases (unless of course something breaks them again in the future). :)
https://rust-lang-nursery.github.io/rust-toolstate/

**Comment by alexcrichton** [comment]
Next issue to be fixed is at https://github.com/rust-lang/rust/pull/56783

**Comment by TimDiekmann** [comment]
> Next issue to be fixed is at #56783

Yay, new nightly!

**Comment by kennytm** [comment]
Closing since nightly is back. Thanks everyone for patience.
